ui/admin/tests/e2e/global-setup.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
+const { chromium } = require('@playwright/test');
+const { checkEnv } = require('./helpers/general');
+
+module.exports = async () => {
+  await checkEnv([
```

Comment:
I love this approach and that solves a problem we have in Desktop client e2e, so I will copy this 😉